docs/src/main/sphinx/installation/cli.rst

```diff
@@ -381,10 +383,39 @@ press :kbd:`Enter`.
 By default, you can locate the Trino history file in ``~/.trino_history``.
 Use the ``TRINO_HISTORY_FILE`` environment variable to change the default.
 
+Batch mode
+----------
+
+Running the Trino CLI with the ``--execute``, ``--file`` or passing queries to
+the standard input uses the batch mode (non-interactive), in which
+the Trino CLI exits after executing all queries, does not report progress,
+and prints results in the ``CSV`` format.
```

Comment:
```suggestion
, and prints results in the ``CSV`` format by default.
```

Reply:
There's a comma at the end of the previous line. The whole sentence describes the default behavior in contrast to the interactive mode, so I'm not sure about adding `by default` at the end. It might confuse people about what default we're talking about.

Reply:
That's a fair point. The table does mention that the output format is configurable.